orders/templates/orders/orders.html

```diff
@@ -0,0 +1,54 @@
+{% extends "main/base_template.html" %}
+{% load static %}
+
+{% block stylesheets %}
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="{% static 'css/orders.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <div class="col-12">
+                <table class="table table-bordered">
+                    <thead>
+                    <tr>
+                        <th scope="col">Name</th>
+                        <th scope="col">Book Date</th>
+                        <th scope="col">Start Date</th>
+                        <th scope="col">End Date</th>
+                        <th scope="col">Status</th>
+                        <th scope="col">Actions</th>
+                    </tr>
+                    </thead>
+
+                    <tbody>
+                    {% for order in orders %}
+                        <tr>
+                            <th scope="row">
+                                {% if user == 'daycare' %}
+                                    {{order.dog_owner_id.name }}
+                                {% else %}
+                                    {{order.daycare_id.name }}
+                                {% endif %}
+                            </th>
+                            <td>{{order.book_date}}</td>
+                            <td>{{order.start_date}}</td>
+                            <td>{{order.end_date}}</td>
+                            <td>{{order.get_order_status}}</td>
+                            <td>
+                                {% if order.is_the_order_approvable %}
+                                    <button type="button" class="btn btn-success">Approve</button>
+                                {% endif %}
```

Comment:
if the method `is_the_order_approvable` is only for daycare as I mentioned, I think you should also check here if the user is a daycare or dog-owner.